gcp/databases/cloudsql/private_postgres/main.tf

```diff
@@ -0,0 +1,45 @@
+data "google_compute_network" "main" {
+  name = var.vpc_network
+  project = var.project_id
+}
+
+module "private-service-access" {
+  source = "git::https://github.com/terraform-google-modules/terraform-google-sql-db.git//modules/private_service_access?ref=v11.0.0"
```

Comment:
`git::https://` is unneccesary , should be able to do `github.com/terraform-google-modules/terraform-google-sql-db//modules/private_service_access?ref=v11.0.0` for github sources ([docs](https://www.terraform.io/language/modules/sources#github))